lib/kafo/configuration.rb

```diff
@@ -57,10 +59,13 @@ def initialize(file, persist = true)
 
     def save_configuration(configuration)
       return true unless @persist
+
+      trimmed = configuration.reject { |key, value| !DEFAULT.key?(key) }
```

Comment:
Isn't this equal, but more efficient?
```suggestion
      trimmed = configuration.slice(*DEFAULT.keys)
```

However, you may want to consider skipping values that are `nil`.

Reply:
Slice for hash was added in Ruby 2.5, we still test against Ruby 2.4 so we have to stick with the current implementation.

Reply:
There must be some joke about sliced bread in there.